src/main/java/net/simpvp/Misc/Restart.java

```diff
@@ -20,6 +21,9 @@
 	private static UUID requester = null;
 	private static long last_request = System.currentTimeMillis();
 
+	// This is a list of players who can't use /cancelrestart
+	List<?> cancel = Misc.instance.getConfig().getList("disableCancelRestart");
```

Comment:
Let's make this list a `List<UUID>` so we can compare the uuids directly instead of as strings.